src/app/markdown/markdown.component.html

```diff
@@ -0,0 +1,53 @@
+ <ul class="nav nav-tabs" *ngIf="editorActive">
+  <li [class.active]="viewType==='markdown'">
+    <a (click)="onClickMarkdownTab()" class="tab-btn">Markdown</a>
+  </li>
+  <li [class.active]="viewType==='preview'">
+    <a (click)="onClickPreviewTab()" class="tab-btn">
+      {{rendering ? "Loading.." : "Preview"}}
+    </a>
+  </li>
+</ul>
+
+<div class="editor-container">
+  <div class="editor"
+    [class.active]="editorActive"
+    [class.show-less]="!showMore && !editorActive">
+    <span (click)="activeEditor()" class="pficon-edit edit-icon"></span>
+    <p almEditable
```

Comment:
Last time when I implemented inputs with textarea in planner I faced some issues with data bindings. Well, I have learned more thing since last time, so I can give it a try and I also feel like editable <p> is not a correct implementation, but let's keep it on an iteration. Shall we?